docs/source/getting_started/loading_data/index.rst

```diff
@@ -122,13 +122,14 @@ to enable the :code:`qt` environment in IPython:
 
 .. code-block:: python
 
-    In[12]: %gui qt
-    In[13]: starfish.display(image)
-    Out[13]: <napari.components._viewer.model.Viewer at 0x15f7b44e0>
+    In[12]: ipython = get_ipython()
+    In[13]: ipython.magic("gui qt5")
+    In[14]: starfish.display(image)
+    Out[14]: <napari.components._viewer.model.Viewer at 0x15f7b44e0>
 
 Typing the above code should display an image viewer that looks something like this:,
 
-#TODO ambrosejcarr annotate an image of the napari viewer.
+.. image:: /_static/design/napari-viewer.png
```

Comment:
I get a 404 on the github rendering of this image. Can you verify that it shows up on the docs you build locally? If that works for you then I think this is good to merge. 👍

Reply:
This is what I get locally
<img width="745" alt="Screen Shot 2019-05-31 at 9 51 41 PM" src="https://user-images.githubusercontent.com/7750862/58741990-62874c80-83ee-11e9-8d81-78323f06f5a1.png">